docs/getting-started/vite-in-sfx.md

```diff
@@ -0,0 +1,80 @@
+<div style="display: flex; justify-content: space-between; align-items: center;">
+
+# Vite in Storefront X
+
+<div style="width: 80px">
+
+![Vite](/assets/images/vite-logo.svg)
+
+</div>
+</div>
+
+Vite is a frontend build tool designed for **fast and efficient development**. It uses an in-memory file system, incremental builds, dynamic imports, a fast development server, and parallel processing to provide fast build times and real-time updates. Vite is also designed to be modular and compatible with popular libraries and frameworks, making it a versatile choice for frontend development.
```

Comment:
I dont think you need to sell Vite that hard. Definitely mention its advantages and disadvantages but I would tone it down a bit. Focus more on what benefits does Vite brings to SFX and why we chose it over anything else.